templates/search_derived/search_clients_last_seen_v1/init.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE
+  search_clients_last_seen_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id,
+  client_id
+OPTIONS
+  ( require_partition_filter=TRUE) AS
+SELECT
+    CAST(NULL AS DATE) AS submission_date,
+
+    -- Grouping columns
+    client_id,
+    CAST(sample_id AS INT64) AS sample_id,
+
+    -- Dimensional data
+    country,
+    app_version,
+    distribution_id,
+    locale,
+    search_cohort,
+    addon_version,
+    os,
+    channel,
+    profile_creation_date,
+    default_search_engine,
+    default_search_engine_data_load_path,
+    default_search_engine_data_submission_url,
+    profile_age_in_days,
+    active_addons_count_mean,
+    user_pref_browser_search_region,
+    os_version,
+
+    -- User activity data
+    max_concurrent_tab_count_max,
+    tab_open_event_count_sum,
+    active_hours_sum,
+    subsession_hours_sum,
+    sessions_started_on_this_day,
+
+    -- Search data
+    organic,
+    sap,
+    unknown,
+    tagged_sap,
+    tagged_follow_on,
+    ad_click,
+    search_with_ads,
```

Comment:
Do we generally want to include all fields from search_clients_daily? Should we express this with a `* EXCEPT(...)` rather than listing out all fields? I assume the exceptions are a much smaller list that would be less likely to change.

Reply:
Reading the main query content, I now realize we have a grouping step where we have to list explicit aggregations for each column, so it makes more sense to me now why we'd list everything in init as well.